modules/instrument_manager/php/NDB_Menu_Filter_instrument_manager.class.inc

```diff
@@ -95,15 +95,21 @@ class NDB_Menu_Filter_Instrument_Manager extends NDB_Menu_Filter
                     null,
                     null
                 );
-
-                exec(
-                    "mysql -u" . escapeshellarg($db_config['quatUser']).
-                    " -p" . escapeshellarg($db_config['quatPassword']).
-                    " " . escapeshellarg($db_config['database']).
-                    " < " . $this->path . "project/tables_sql/".
-                    $instrument->table.".sql"
-                );
-
+		if(!isset($db_config['quatUser'], $db_config['quatPassword'])){
+			throw new LorisException(
+				"SQL needs to be sourced manually at /project/tables_sql/".
+				$instrument->table.".sql"
+			);
+		} 
+	        exec(
```

Comment:
did not know that was a thing, just did it now and the indentation is all fixed!